examples/Components/collision/RuleBasedContactManager.scn

```diff
@@ -65,7 +65,7 @@
 		<EulerImplicit />
 		<CGLinearSolver iterations="25" tolerance="1e-5" threshold="1e-5"/>
         <MechanicalObject template="Rigid" scale="5.0" dx="-38" dy="0" />
-        <UniformMass filename="BehaviorModels/torus.rigid" totalMass="5000.0" />
+        <UniformMass filename="BehaviorModels/torus.rigid"/>
```

Comment:
for me uniformMass  was applying a value: totalMass/nbrVertex for each point. What data is this file giving?